src/app/pages/contract-dashboard/contract-dashboard.component.html

```diff
@@ -1 +1,45 @@
-<p>contract-dashboard works!</p>
+<mat-form-field appearance="outline" style="width:40vw; font-size: 12px" color="warning">
+  <mat-label>Search</mat-label>
+  <input matInput placeholder="Search">
+</mat-form-field>
+<button mat-flat-button color="primary" class = "searchbutt">Search</button>
+<button mat-flat-button  color="primary" class = "filterbutt" >Filter</button>
+<button mat-flat-button  color="primary" class = "createbutt">Create Contract</button>
+<table mat-table [dataSource]="contracts" class="mat-elevation-z8 table">
+
+  <ng-container matColumnDef="vendor_name">
+    <th mat-header-cell *matHeaderCellDef> Name </th>
+    <td mat-cell *matCellDef="let element">{{element.vendor_name}}</td>
+  </ng-container>
+
+ <ng-container matColumnDef="date_range">
+    <th mat-header-cell *matHeaderCellDef> Dates </th>
+    <td mat-cell *matCellDef="let element">{{element.date_range}}</td>
+  </ng-container>
+
+  <ng-container matColumnDef="amount_due">
+    <th mat-header-cell *matHeaderCellDef> Amount due </th>
+    <td mat-cell *matCellDef="let element">{{element.amount_due}}</td>
+  </ng-container>
+
+  <ng-container matColumnDef="paid">
+    <th mat-header-cell *matHeaderCellDef> Paid? </th>
+    <td mat-cell *matCellDef="let element">{{element.paid}}</td>
+  </ng-container>
+
+  <ng-container matColumnDef="work_id">
+    <th mat-header-cell *matHeaderCellDef> Work Type </th>
+    <td mat-cell *matCellDef="let element">{{element.work_id}}</td>
+  </ng-container>
+
+  <ng-container matColumnDef="butt">
+    <th mat-header-cell  *matHeaderCellDef > Edit </th>
+    <td mat-cell *matCellDef="let row" >
+         <button mat-flat-button class = "editbutt">Edit</button>
```

Comment:
Add payment is a button, plaiing on adding line click functionality as soon as the contract form is available